pulp_container/app/downloaders.py

```diff
@@ -204,6 +205,12 @@ def _make_aiohttp_session_from_remote(self):
         """
         tcp_conn_opts = {"force_close": True}
 
+        if not self._remote.tls_validation:
```

Comment:
Are you sure this is the full fix? For the sigstore source that would suffice.
I see discrepancy in the code here https://github.com/pulp/pulp_container/blob/main/pulp_container/app/tasks/sync_stages.py#L273 and here https://github.com/pulp/pulp_container/blob/main/pulp_container/app/tasks/sync_stages.py#L605
I would expect same downloader to be used for the extension api signature source discovery as well as for the signature download from the same registry source.

Reply:
Good find. Was there any reason for having a separate downloader for each case? I can get rid of the discrepancy in any case.

Anyway, the issue should be resolved. In `get_downloader`, we use the facility from `DownloaderFactory` that does handle the TLS validation option automatically.

Reply:
I am not sure there was any reason. I think this falls, rather, under the bug/oversight category.

Reply:
The question is if we should have a separate remote for signatures. It seems to me that using a single remote might be conceptually wrong.

Reply:
It depends. Having same remote for signature extensions api is fine since same remote setting will be used. For the separate sigstore location only in case that one has it's own config. Since we work from the assumption that the separate sigstore does not have any auth, there is no need to provide a separate remote object. If things will change in the future we can address them then but right now we have limited hands-on experience with separate sigstore location except for red hat sigstore.